app/routes/circulars/index.tsx

```diff
@@ -55,17 +55,20 @@ export async function loader({ request: { url } }: DataFunctionArgs) {
 
 function getPageLink({
   page,
+  limit,
   query,
   startDate,
   endDate,
 }: {
   page: number
+  limit?: string
```

Comment:
Please follow the example of the `page` argument here and accept the `limit` argument as a number.

```suggestion
  limit?: number
```

Reply:
converted to number type in [2487eb6](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1104/commits/2487eb68d08cd1ecbf7fb90d62dd4a41b3d7142a)